doc/devel/coding_guide.rst

```diff
@@ -63,7 +63,19 @@ PR Review guidelines
   :file:`doc/api/api_changes` and significant new features have and
   entry in :file:`doc/user/whats_new`.
 
-* Make sure the Travis, Appvyor, circle, and codecov tests are passing
+  - If a PR already has a positive review, a core developer (e.g. the first
+    reviewer, but not necessarily) may champion that PR for merging.  In order
+    to do so, they should ping all core devs both on github and on the dev
+    mailing list, and label the PR with the "Merge with single review?" label.
```

Comment:
I'm not sure about "_and_ on the dev mailing list"; I rarely see day-to-day pings (as opposed to long-term, more architectural things) happen there.

Reply:
I think it doesn't cost much to send an email.

Reply:
I mean that I don't see it happening regularly...

Reply:
I'm fine either way, but let's see what @tacaswell thinks?